dev/release/README.md

```diff
@@ -0,0 +1,474 @@
+    $ gpg2 --detach-sign -a apache-spot-1.0-incubating..tar.gz
+ 
+
+#### **Retrieve the subversion dev incubator repo for SPOT**
+
+Example:
+* svn checkout https://dist.apache.org/repos/dist/dev/incubator/spot/ --username=`<your apache user>`
+ 
+Create a local folder for the release (e.g. 1.0-incubating) in svn. 
+* svn mkdir -m "Creating SPOT `<release number>` dir" https://dist.apache.org/repos/dist/dev/incubator/spot/`<release number>` --username=`<your apache user>`
+
+Example:
+
+    svn mkdir -m "Creating SPOT 1.0-incubating dir" https://dist.apache.org/repos/dist/dev/incubator/spot/1.0-incuabting --username=`<your apache user>`
+
+
+Move the files into the release folder on local disk.
+* svn add `<release folder>`
+
+Example:
+
+    svn add 1.0-incubating/
+
+Commit artifacts:
+* svn commit -m '`<custom message>`' --username=`<your apache user id>`
+
+Example:
+
+    svn commit -m 'adding spot 1.0-incubating candidate release artifacts' --username=`<your apache user id>`
+
+## **Validate the Build**
+
+Download the tarball.
+
+     http://spot.apache.org/download
+
+Decompress the tarball. Instruction:
+    
+    tar -zxvf apache-spot-1.0-incubating.tar.gz
+
+Change directory. Instruction:
+    
+    cd apache-spot-1.0-incubator
+
+Apache Spot (incubating) is composed of more than one module or sub-projects. Since some of them are Python or Javascript code, they don’t need compilation.
+
+For more instructions about how to install each module please read below instructions.
+
+ You should see the content of the folder:
+
+    spotadmin-mac01:apache-spot-1.0-incubating spotadmin$ ls -la
+    total 72
+    drwxr-xr-x 14 spotadmin staff 476 Jul 24 16:45 .
+    drwxr-xr-x 7 spotadmin staff 238 Aug 4 09:32 ..
+    -rw-r--r-- 1 spotadmin staff 20 Jul 24 16:45 .gitignore
+    -rw-r--r-- 1 spotadmin staff 0 Jul 24 16:45 .gitmodules
+    -rw-r--r-- 1 spotadmin staff 560 Jul 24 16:45 DISCLAIMER
+    -rw-r--r-- 1 spotadmin staff 11918 Jul 24 16:45 LICENSE
+    -rw-r--r-- 1 spotadmin staff 1493 Jul 24 16:45 LICENSE-topojson.txt
+    -rw-r--r-- 1 spotadmin staff 159 Jul 24 16:45 NOTICE
+    -rw-r--r-- 1 spotadmin staff 6761 Jul 24 16:45 README.md
+    drwxr-xr-x 3 spotadmin staff 102 Jul 24 16:45 docs
+    drwxr-xr-x 10 spotadmin staff 340 Jul 24 16:45 spot-ingest
+    drwxr-xr-x 13 spotadmin staff 442 Jul 24 16:45 spot-ml
+    drwxr-xr-x 11 spotadmin staff 374 Jul 24 16:45 spot-oa
+    drwxr-xr-x 10 spotadmin staff 340 Jul 24 16:45 spot-setup
+
+Decompressed tarball content should be the same with the content located in: 
+
+* [https://github.com/apache/incubator-spot/tree/v1.0-incubating](https://github.com/apache/incubator-spot/tree/v1.0-incubating)
+
+To install the properly component please follow this guide:
+
+* [http://spot.apache.org/doc/#installation](http://spot.apache.org/doc/#installation)
+
+Spot-ingest, Spot-setup, Spot UI and Spot-OA have specific requirements to install manually.
```

Comment:
Do not use _Spot-_ when not referring to folders in the project. For a _formal_ name of the module use camel case i.e. Spot Ingest, Spot OA, Spot ML and Spot UI instead of Spot-ingest, Spot-OA, Spot-oa, etc.